mettle/extensions/sniffer/README.md

```diff
@@ -0,0 +1,79 @@
+SNIFFER EXTENSION
+=================
+
+This loadable extension allows users to capture network traffic on a target.
+
+Loading the Sniffer
+-------------------
+
+Once you've established a connection to Mettle, you can load the sniffer extension with the following command in your msfconsole terminal:
+
+`load sniffer`
+
+If things go according to plan, you should get a message that the extension was successfully loaded:
+
+```
+meterpreter > load sniffer
+Loading extension sniffer...Success.
+```
+
+Using the Sniffer
+-----------------
+
+The sniffer commands operate on a similar paradigm as the keylogger extension commands do.  You can see the available commands with `help` in your msfconsole terminal:
+
+```
+Sniffer Commands
+================
+
+    Command             Description
+    -------             -----------
+    sniffer_dump        Retrieve captured packet data to PCAP file
+    sniffer_interfaces  Enumerate all sniffable network interfaces
+    sniffer_release     Free captured packets on a specific interface instead of downloading them
+    sniffer_start       Start packet capture on a specific interface
+    sniffer_stats       View statistics of an active capture
+    sniffer_stop        Stop packet capture on a specific interface
+```
+
+An Example
+----------
+
+In this example, I will load the sniffer extension on my target MacBook and then do the following:
+
+* list the available network interfaces
+* start sniffing on an interface
+* through the sniffed interface I will:
```

Comment:
Maybe something more like "on the target, the user will" as to not confuse what you the attacker is doing vs what the victim is doing